File: Interviews/LeetCode/Topic-Wise/Strings/Easy/Check-if-the-Sentence-Is-Pangram/1832.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        cnt = [0 for i in range(26)]
        for c in sentence:
            cnt[ord(c) - 97] |= 1
        for i in cnt:
            if not i:
                return False
        
        return True

File: Interviews/LeetCode/Daily-Problems/2022/October/Day-17/1832.cpp
class Solution {
public:
    bool checkIfPangram(string sentence) {
        vector<int> cnt(26, 0);
        for (char c : sentence) {
            cnt[c - 'a'] |= 1;
        }

        for (int i = 0; i < 26; i++) {
            if (!cnt[i]) {
                return false;
            }
        }

        return true;
    }
};